### PR TITLE
Add Screensaver Ninja as a Mac OS X tool to display Travis CI.

### PR DESCRIPTION
**Pull request by pupeno**

### CODE REVIEW

File: user/apps.md [hunk 1]
```diff
@@ -292,6 +292,16 @@ By ThoughtWorks Inc.
 * [app store](https://itunes.apple.com/us/app/ccmenu/id603117688?mt=12&ign-mpt=uo%3D4)
 * [tutorial](/user/cc-menu/)
 
+![Travis CI in Screensaver Ninja with Custom CSS](/images/apps/screensaver-ninja.gif){:.app}
```
Comment (BanzaiMan) [inline user/apps.md]: This screenshot should appear below the "Screensaver Ninja" heading.

Comment (pupeno) [inline user/apps.md]: I can change it, but that's not true for CCMenu, not sure about others, but the next entry, which is for Linux, seems to also have the screenshot above the heading. Do you want me to still change it?

Comment (BanzaiMan) [inline user/apps.md]: Oh! You are right about that.

Comment (pupeno) [inline user/apps.md]: Do you need me to do anything else here?

**Comment by plaindocs** [comment]
@BanzaiMan @lislis is looking into the formatting on this page. If this is the same as the others I'd say just merge it in.

**Pull request merged**